multi_physics/QED/python_bindings/CMakeLists.txt

```
@@ -0,0 +1,64 @@
+set(name pxr_qed)
+
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+find_package(pybind11 CONFIG REQUIRED)
```

Let's add a minimal version here, e.g. the new logic for CMake came in in 2.6+ https://github.com/pybind/pybind11/releases
```suggestion
find_package(pybind11 2.6.0 CONFIG REQUIRED)
```